golang/cmd/crane/crane.go

```diff
@@ -46,41 +45,44 @@ func main() {
 	}
 }
 
-func loadConfiguration() (*config.Configuration, error) {
+func loadConfiguration() (*config.Configuration, *k8s.Secret, error) {
 	cfg := &config.Configuration{}
+
 	err := util.ReadConfig(cfg)
 	if err != nil {
 		log.Panic().Err(err).Msg("Failed to load configuration")
 	}
-	if err = cfg.ParseAndSetJWT(os.Getenv("GRPC_TOKEN")); err != nil {
-		log.Panic().Err(err).Msg("Failed to parse env GRPC_TOKEN")
-	}
+
 	client := k8s.NewClient(cfg)
 	secretHandler := k8s.NewSecret(context.Background(), client)
-	secret, err := secretHandler.GetValidSecret()
+
+	cfg.InjectPrivateKey(secretHandler)
 	if err != nil {
-		return nil, err
+		log.Panic().Err(err).Msg("Failed to load private key")
 	}
 
-	commonConfig.InjectSecret(secret, &cfg.CommonConfiguration)
+	err = cfg.InjectGrpcToken(secretHandler)
+	if err != nil {
+		log.Panic().Err(err).Msg("Failed to load grpc token")
```

Comment:
gRPC

Reply:
I think here the camel case is more dominant. @nandor-magyar ?

Reply:
![image](https://github.com/dyrector-io/dyrectorio/assets/9247788/2a209506-0500-4d9c-ac40-579186e8dda9)
proof

Reply:
I can see that, I'm not saying this isn't how google prefers to write grpc. I am saying this should not affect the function names, as coding conventions has higher priority than marketing ones.

Reply:
The comment is against a string, not a variable name, in this case gRPC it is, otherwise in function names, yes. Once again, I'm 100% sure this was **directed** towards the message.

Reply:
Ah, silly me. I didn't noticed that. Sorry!